virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -510,3 +510,30 @@ def check_in_vm(vm, target, old_parts, is_equal=False):
     except Exception as e:
         logging.error(str(e))
         return False
+
+
+def create_remote_disk_by_first(vm, params):
+    """
+    Create an empty image on remote host
+
+    :param vm:  the VM object migrated
+    :param params:  dict, parameters used for migration
```

Comment:
Should we remove 'migrated' and 'migration'?

Reply:
updated